app/src/screens/ProposalDetail.js

```diff
@@ -1,15 +1,16 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import {
   BackButton,
   Bar,
   Box,
   GU,
   Text,
   textStyle,
-  useLayout,
-  useTheme,
   Link,
+  SidePanel,
   Split,
+  useLayout,
+  useTheme,
 } from '@aragon/ui'
 import styled from 'styled-components'
```

Comment:
should we remove the styled import from here and use the tags with `css`?

Reply:
Yeah let's do it

Reply:
I think we can do it for the `Chart` component, but for the `H2` probably is best to leave it as a styled-component since it's used several times.